napalm_ios/ios.py

```diff
@@ -27,6 +27,7 @@
 from napalm_base.exceptions import ReplaceConfigException, MergeConfigException
 from napalm_base.utils import py23_compat
 import napalm_base.constants as C
+import napalm_base.helpers
```

Comment:
I don't the the use of the helpers is very consistent... Not very worried about it though. We can fix things as we go along and OC (which should be the way forward) has type checking builtin. I know it sounds very optimistic but we are doing a huge work here so there is no point of despairing : )